app/inpututils.cpp

```diff
@@ -2013,3 +2013,33 @@ QString InputUtils::imageGalleryLocation()
 
   return galleryPaths.last();
 }
+
+QList<QgsPoint> InputUtils::parsePositionUpdates( const QString &data )
+{
+  QList<QgsPoint> parsedUpdates;
```

Comment:
Any reason why QList and not QVector is used?

Reply:
I think QVector is just an alias for QList in Qt6 
<img width="399" alt="image" src="https://github.com/MerginMaps/input/assets/22449698/d2f4e810-91a1-4fd4-988f-4f2d682f06be">

Reply:
https://doc.qt.io/qt-6/qvector.html

Reply:
good to know. thanks